charts/charts/tenant-namespace/values.yaml

```diff
@@ -88,6 +88,20 @@ ingress:
   rbac:
     scope: true
 
+gateway:
+  enabled: false
+  name: public
+  className: public
+  listeners: []
+#    - allowedRoutes:
```

Comment:
since listeners can be overridden with a single entry (list behavior), should we make it default listen on 80/443?